src/controllers/action-queue/mock.ts

```diff
@@ -93,11 +92,6 @@ async function checkActionEffect(account: EdgeAccount, effect: ActionEffect): Pr
         isEffective: true
       }
     }
-    default: {
```

Comment:
Add the default case back and throw

Reply:
The `default` case never seen the light of runtimey day. It was always a type checking hack.